### ISSUE TITLE
Choose Decryption Key Import Format

### ISSUE THREAD

**Issue by acnicholls** [acnicholls/Cypher#12]
Need to choose a format to store "decryption keys"

once the feature for variable alphabets is included and the default value for decryption is set to unknown the decryption process on standard machines is going to exponentially grow in length of time, since the number of permutations will be multiplied by a factor of 26 (1 for each version of the "numerical alphabet" ).

Since the feature also includes the ability to SET an alphabet in the encryption process, that SAME alphabet can be used to assist in decryption, like a key.  The application will still be required to create and process each permutation, but there will be exponentially less permutations to decrypt since the alphabet values used to encrypt it will be known.

basically needs to store an array of 26 numbers, maybe with keys for ease of reading the data?
JSON
CSV
TSV
XLS
XLSX
SSV
